packages/react-native/js-modules.ios.ts

```diff
@@ -0,0 +1,26 @@
+import { getCurrentBridge } from './bridge.ios';
+import { toJSValue } from './converter.ios';
+
+class JSModules {
+  _bridge: RCTBridge;
+  _modules: { [name: string]: any };
+  constructor() {
+    this._bridge = getCurrentBridge();
+    this._bridge.setJSModuleInvokerCallback(this.jsModuleInvoker);
+    this._modules = {};
+  }
+
+  jsModuleInvoker(moduleName: string, methodName: string, args: NSArray<any>) {
+    this._modules[moduleName]?.[methodName]?.(...toJSValue(args));
+  }
+
+  registerJSModule(name: string, module: any) {
+    this._modules[name] = module;
+  }
+
+  unregisterJSModule(name: string) {
+    delete this._modules[name];
+  }
```

Comment:
Removing the `_` from the private members.
I've also returned the value of the invoked JS module - might be relevant for functions that synchronously return a value.
```suggestion
  jsModuleInvoker(moduleName: string, methodName: string, args: NSArray<any>) {
    return this.modules[moduleName]?.[methodName]?.(...toJSValue(args));
  }

  registerJSModule(name: string, module: any) {
    this.modules[name] = module;
  }

  unregisterJSModule(name: string) {
    delete this.modules[name];
  }
```